client/api/volume/v1beta1/api.proto

```diff
@@ -91,6 +91,8 @@ message ResizeVolumeResponse {
 message VolumeStatsRequest{
     // Volume device Id of the volume to get the stats for
     string volume_id = 1;
+    // Path where the volume has been mounted.
+    string path = 2;
```

Comment:
I think we should make a v1beta2 if we modify the API in any way. Because csi proxy versions are not tied to K8s versions, it is not possible to tell if the version of csi proxy running on the node supports this new field or not.

Reply:
agree. updated to v1beta2. PTAL